sct.py

```diff
@@ -1117,6 +1119,9 @@ def send_email(test_id=None, test_status=None, start_time=None, started_by=None,
                     "nodes": get_running_instances_for_email_report(test_id, runner_ip)
                 })
         test_results['logs_links'] = list_logs_by_test_id(test_results.get('test_id', test_id))
+        if 'longevity' in job_name:
```

Comment:
Why we can look for it for any job ?

Reply:
I think we can. But since we use the Parallel Timeline for longevities only I just wanted to avoid some additional S3 scanning for all other jobs. Should I rework this part?